src/main/java/net/dv8tion/jda/internal/entities/ActivityImpl.java

```diff
@@ -123,9 +123,6 @@ public int hashCode()
     @Override
     public String toString()
     {
-        if (url != null)
-            return String.format("Activity(%s | %s)", name, url);
-        else
-            return String.format("Activity(%s)", name);
+        return "Activity[" + name + (url != null ? "](" + url + ')' : ']');
```

Comment:
```suggestion
        if (url != null)
            return String.format("Activity[%s](%s)", name, url);
        else
            return String.format("Activity(%s), name);
```
The original form was easier to read.